utils/auc/src/auc.c

```diff
@@ -66,6 +67,9 @@
 
 static const char server_issues[]="https://github.com/openwrt/asu/issues";
 
+static const char uci_defaults_filename[] = "/rom/etc/uci-defaults/99-asu-defaults";
+static const int uci_defaults_size_max = 10240;
```

Comment:
I was digging through the ASU server code and noticed that the buffer size for `defaults` was increased to 20k some time ago (the documentation string was not updated, it incorrectly states "10kB").

https://github.com/openwrt/asu/commit/d3883b05d22474b1032953dcdbf2b9aec556b88b

Reply:
Thank you
https://github.com/openwrt/asu/pull/693

Reply:
done